qml/plugins/RestartBluetooth.qml

```diff
@@ -0,0 +1,17 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.Utilities 1.0
+
+ActionItem {
+    //% "Bluetooth"
+    title: qsTrId("settings_bluetooth-bluetooth")
+    //% "Restart"
+    actionName: qsTrId("sailfish-tools-bt-restart")
+    deviceLockRequired: false
+    //% "Restart Bluetooth subsystem"
+    description: qsTrId("sailfish-utilities-me-restart-bt-desc")
```

Comment:
```suggestion
    description: qsTrId("sailfish-utilities-me-restart_bluetooth_desc")
```
Would something like this be okay?

Reply:
The other qmls (e.g. restart netwoking) have dashes not underscores for these. What is the convention here?

Reply:
Underscoring after placement code, like `myapp-la-localized_label`. You're right that other qmls have dashes all over but it's not worth to change those. Changing those would make them to appear as new translation strings in the translation tool.

- myapp => name of the component
- la => placement code, in this case label
- localized_label, descriptive text of the id with underscores

Reply:
Thanks. Done.